core/src/main/java/org/neo4j/ogm/session/delegates/ExecuteQueriesDelegate.java

```diff
@@ -69,7 +69,8 @@
  */
 public class ExecuteQueriesDelegate extends SessionDelegate {
 
-    private static final Pattern WRITE_CYPHER_KEYWORDS = Pattern.compile("\\b(CREATE|MERGE|SET|DELETE|REMOVE|DROP|CALL)\\b");
+    private static final Pattern WRITE_CYPHER_KEYWORDS = Pattern.compile("\\b(CREATE|MERGE|SET|DELETE|REMOVE|DROP|CALL)\\b",
+        Pattern.CASE_INSENSITIVE | Pattern.UNICODE_CASE);
```

Comment:
Nice one! I expected the regex op before the regex, didn't knew about the `UNICODE_CASE` modifier.